docs/reference/release-notes/860.md

```diff
@@ -113,6 +113,18 @@ On Camunda 8 SaaS, you can now configure hourly backup intervals. Simply head to
 
 In Operate, the visualization of the executed path of a process instance is improved. Both executed sequence flows and flow nodes are highlighted, allowing you to display how often each flow node was executed.
 
+### Write rate limiting based on exporting rate
+
+<!-- https://github.com/camunda/product-hub/issues/1426 -->
+
+We introduce a new unified flow control mechanism inside of Zeebe that is able to limit user commands (by default it tries to achieve 200ms response times) and rate limit writes of new records in general.
+
+Limiting the write rate is a new feature that can be used to prevent building up an excessive exporting backlog. There are two ways to limit the write rate, either by setting a static limit or by enabling throttling that dynamically adjust the write rate based on the exporting backlog and rate. If activated you might see more back pressure in cases where exporters are not exporting fast enough a backlog builds up. The back pressure error message will contain information to explain what is happening and you can read documentation to better understand what to do in such cases.
+
+We apply the new write limit in SaaS to our default hosting packages automatically.
```

Comment:
Non-blocking suggestion (reword):

The new write limit is automatically applied to Saas default hosting packages.